conf/ocs_version/ocs_4_2.yaml

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
we can remove the old ocs_4.2 conf file ?

Reply:
I pulled these from @shylesh 's WIP PR, but I think I actually need to update them. I will pull in the values from the file @dahorak added for 4.3 and create a similar one for 4.2 (using our default values).

Reply:
We have one for 4.3 that we can remove. I just wanted to verify that the values in the files I created are correct before removing, but I will do that as a part of this PR.

Reply:
Removed the old ocs 4.3 file

Reply:
Actually holding off on deleting this until we are sure it's not being used in any automation.